pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
   "tables",
   "torch",
   "tiled",
+  "xrt",
```

Comment:
Can we move this to an optional dependency? Similar to sirepo

Reply:
will do!